integration-tests/src/test/java/gov/uk/address/api/stepdefinitions/AddressSteps.java

```diff
@@ -117,41 +126,64 @@ public void aValidJWTIsReturnedInTheResponse() throws ParseException, IOExceptio
         makeAssertions(SignedJWT.parse(this.testContext.getResponse().body()));
     }
 
-    @Then("TXMA event is added to the SQS queue containing device information header")
-    public void txmaEventIsAddedToSqsQueueContainingDeviceInformationHeader()
-            throws IOException, InterruptedException {
-        assertEquals("deviceInformation", getDeviceInformationHeader());
-    }
+    @And("a valid START event is returned in the response with txma header")
+    public void aValidStartEventIsReturnedInTheResponseWithTxmaHeader() throws IOException {
+        String responseBody = testContext.getTestHarnessResponseBody();
+        List<TestHarnessResponse<AuditEvent<Map<String, Object>>>> events =
+                objectMapper.readValue(responseBody, new TypeReference<>() {});
 
-    @Then("TXMA event is added to the SQS queue not containing device information header")
-    public void txmaEventIsAddedToSqsQueueNotContainingDeviceInformationHeader()
-            throws InterruptedException, IOException {
-        assertEquals("", getDeviceInformationHeader());
-    }
+        assertFalse(events.isEmpty());
```

Comment:
Could we assert the amount of events instead of just empty, e.g. if 2 duplicate start events were sent it would flag up then